teamA/initial-settings-express/heeyeon/app.js

```diff
@@ -0,0 +1,41 @@
+const dotenv = require('dotenv');
+dotenv.config();

```

Comment:
해당 코드는 한 줄로 줄여서 사용할 수 있을 것 같네요!

```suggestion
require('dotenv').config()
```